Jenkinsfile

```diff
@@ -37,14 +37,28 @@ pipeline {
       environment {
         CF_APP_NAME="portaljs${env.CF_SPACE == 'production' ? '' : '-' + env.CF_SPACE}"
         CTF_CPA_ACCESS_TOKEN=credentials("portaljs.${env.CF_SPACE}.contentful.cpa")
+        LOGSTASH_CONNECTION=credentials("elk_connection_string")
         HTTP_DIGEST_ACL=credentials("portaljs.${env.CF_SPACE}.http.digest.acl")
       }
       steps {
+        sh 'if [[ "${LOGSTASH_CONNECTION}" != "" ]]; then sed -i "s|services:|services:\\n  - log-drainer|" manifest.yml; fi'
```

Comment:
I would be inclined to put the name of the log-drainer service into a global env var in Jenkins too, named `CF_LOG_DRAINER_SERVICE_NAME`